onnxruntime/test/providers/cpu/activation/activation_op_test.h

```diff
@@ -69,6 +69,16 @@ inline void TestActivationOp(const char* szOp, const std::vector<std::vector<T>>
     }
 #endif
 
+#if defined(USE_NUPHAR)
+ if (!std::is_same<T, float>::value) {
+    int sigmoid = strcmp(szOp, "Sigmoid");
+    int tanh = strcmp(szOp, "Tanh");
+    // nuphar implementation uses float constants which does not work with other types.
+    if (sigmoid == 0 || tanh == 0) {
+      excluded_providers.insert(kNupharExecutionProvider);
```

Comment:
seems like Nuphar EP should not register Sigmoid/Tanh kernels for non-float type?